frontend/src/app/projects/project-detail/project-metadata/project-metadata.component.ts

```diff
@@ -6,20 +6,15 @@
 import {
```

Comment:
There is a small issue, to reproduce: 

1. Open a project from the project overview
2. The "Submit" button is disabled
3. One wants to change the description
4. The "Submit" button is still disabled, so it can not be submitted.

Reply:
Yes, this is a "common" problem that I also encountered when simplifying the `edit-git-instance` thing. The problem here is that the slug validator throws an error because the project slug already exists, so the form is not valid. The simplest and also a good solution to this problem is to pass a project to the asynchronous validator, which is then excluded/ignored. I have applied this and now it works.